index.html

```diff
@@ -65,38 +67,104 @@ <h2 class="my-name">I'm Yemsrach Glad to see you!</h2>
     </div>
     </div>
     </section>
-  
-    <section class="work">
-       <section class="work">
+    <!-- Home section start -->
+  <!-- wotk section start -->
+  <section class="work grid-container">
 <div class="snapshoot">
   <img src="./image/Snapshoot Portfolio.png" alt="snapchat">
-  <div class="snapshoot_1"></div>
-  <h1>Tonic</h1>
+  <div class="snapshoot_1">
+  <h2>Tonic</h2>
   <ul class="profile">
   <li>CANOPY</li>
   <li><img src="./image/Ellipse 5.png" alt="ellipse"></li>
   <li>Back End Dev</li>
   <li><img src="./image/Ellipse 5.png" alt="ellipse"></li>
   <li>2015</li>
   </ul>
+</div>
   <p>A daily selection of privately presonalizaed reads: no accounts or singn-up required.</p>
   <ul class="skill">
-    <li><h4>html</h4></li>
-  <li>css</li>
-  <li>javaScript</li>
+    <li><button>html</button></li>
+  <li><button>css</button></li>
+  <li class="javaScript"><button>javaScript</button></li>
   </ul>
-  <button class="btn"><a href="#">See Project</a></button>
+  <a class="see_project" href="#"><button>See Project</button></a>
+</div>
+<div class="snapshoot">
+  <img src="./image/Snapshoot Portfolio (1).png" alt="snapchat">
+  <div class="snapshoot_1">
+  <h2>Multi_Post Stories</h2>
+  <ul class="profile">
+  <li>CANOPY</li>
+  <li><img src="./image/Ellipse 5.png" alt="ellipse"></li>
+  <li>Back End Dev</li>
+  <li><img src="./image/Ellipse 5.png" alt="ellipse"></li>
+  <li>2015</li>
+  </ul>
+</div>
+  <p>A daily selection of privately presonalizaed reads: no accounts or singn-up required.</p>
+  <ul class="skill">
+    <li><button>html</button></li>
+  <li><button>css</button></li>
+  <li  class="javascript"><button>javaScript</button></li>
+  </ul>
+  <a class="see_project" href="#"><button>See Project</button></a>
+</div>
+<div class="snapshoot">
+  <img src="./image/Snapshoot Portfolio (2).png" alt="snapchat">
+  <div class="snapshoot_1">
+  <h2>Tonic</h2>
+  <ul class="profile">
+  <li>CANOPY</li>
+  <li><img src="./image/Ellipse 5.png" alt="ellipse"></li>
+  <li>Back End Dev</li>
+  <li><img src="./image/Ellipse 5.png" alt="ellipse"></li>
+  <li>2015</li>
+  </ul>
+</div>
+  <p>A daily selection of privately presonalizaed reads: no accounts or singn-up required.</p>
+  <ul class="skill">
+    <li><button>html</button></li>
+  <li><button>css</button></li>
+  <li class="javaScript"><button>javaScript</button></li>
+  </ul>
+  <a class="see_project" href="#"><button>See Project</button></a>
+</div>
+<div class="snapshoot">
+  <img src="./image/Snapshoot Portfolio (3).png" alt="snapchat">
+  <div class="snapshoot_1">
+  <h2>Multi-Post Story</h2>
+  <ul class="profile">
```

Comment:
[OPTIONAL] Kindly reduce the width and give your image some padding so that the styling can be in line with the Figma design.

**Your design**
<img width="363" alt="image" src="https://user-images.githubusercontent.com/73854999/205436886-3870f370-284c-48be-bc8e-7011cb370f02.png">

**Figma design**
<img width="363" alt="image" src="https://user-images.githubusercontent.com/73854999/205436895-02dc9d76-ab32-4a12-962f-0b68364a8ecd.png">

Reply:
Fixed.